### PR TITLE
Simple selfrando check support

### PR DESCRIPTION
**Pull request by Estella**
https://github.com/immunant/selfrando

Likely better in extended checking?

### CODE REVIEW

File: checksec [hunk 1]
```diff
@@ -37,6 +38,13 @@
 #
 # --- Modified Version ---
 # Name    : checksec.sh
+# Version : x.x.x (selfrando)
+# Author  : Estella Mystagic
```
[the remark this answers is not in the document]
Comment (Estella) [reply checksec]: Oh you can remove those parts, I just wanted to have the check for selfrando added, if you want it could be in extended

**Comment by slimm609** [comment]
added in

**Comment by Maryse47** [comment]
@Estella @slimm609 could you make selfrando check optional like CFI is? This is rare feature which you won't find on any common distro and right now it wastes some space in output

**Comment by slimm609** [comment]
Yea, that should be easy to do

**Comment by slimm609** [comment]
2.2.2 has it moved to extended checks

**Pull request closed without merging**